index.js

```diff
@@ -55,7 +56,7 @@ class ConfirmHcaptcha extends PureComponent {
         onBackdropPress={() => this.hide('backdrop')}
         onBackButtonPress={() => this.hide('back_button')}
         isVisible={show}
-        hasBackdrop={!passiveSiteKey}
+        hasBackdrop={hasBackdrop}
```

Comment:
Not to change the current behavior, please try out the following:
```
hasBackdrop={!passiveSiteKey && hasBackdrop}
```

Reply:
Done! and updated the README